genienlp/ned/ned_utils.py

```diff
@@ -281,18 +281,18 @@ def init_ned_model(args, ned_class_name=None):
     ned_model = None
     if args.do_ned:
         if ned_class_name is None:
-            if args.ned_retrieve_method == 'bootleg':
+            if args.ned_class_name == 'bootleg':
                 ned_class_name = 'Bootleg'
-            elif args.ned_retrieve_method == 'bootleg-annotator':
+            elif args.ned_class_name == 'bootleg-annotator':
```

Comment:
yes. that's why there's the option of passing `ned_class_name` as an argument to override what args say.
Alternatively, we can pass a `mode` which can be (server|batch) but I avoided that since server mode and batch mode for non bootleg classes is the same.